### PR TITLE
Fix Rainbow cosmetics name formatting

### PR DESCRIPTION
**Pull request by dennishzg**
### What is the purpose of this pull request?

1. Fix the formatting of the RainbowWings name. "Rainbow Wings" isn't bold.
2. RainbowWings and Rainbow projectile effect should have the same colors.

Screenshot (26)

### CODE REVIEW

File: core/src/main/resources/messages/messages_en.yml [hunk 1]
```diff
@@ -461,7 +461,7 @@ Particle-Effects:
     name: '&f&lAngel Wings'
     Description: '&7&oBecome an angel!'
   RainbowWings:
-    name: '&l&cR&6a&ei&an&9b&1o&dw&f Wings'
+    name: '&c&lR&6&la&e&li&a&ln&9&lb&1&lo&d&lw &f&lWings'
```
Comment (SirArkimedes) [inline core/src/main/resources/messages/messages_en.yml:464]: Should `Rainbow` be the same color order as the Projectile one that was created first (by you 😄)?
```suggestion
    name: '&4&lR&6&la&e&li&a&ln&b&lb&1&lo&5&lw &f&lWings'
```

**Review by SirArkimedes — APPROVED** [review]
Sorry about that! Thanks for updating it. haha

File: core/src/main/resources/messages/messages_en.yml (hunk 1 above)
Comment (dennishzg) [reply core/src/main/resources/messages/messages_en.yml]: Yes, fixed. 👍

**Comment by datatags** [comment]
Looks good, thanks!

**Pull request merged**